apps/spruce/src/pages/adminSettings/tabs/GeneralTab/ExternalCommunicationsTab/getFormSchema.ts

```diff
@@ -0,0 +1,114 @@
+import { GetFormSchema } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import { objectGridCss } from "../../sharedStyles";
+import {
+  cedar,
+  fws,
+  jira,
+  runtimeEnvironments,
+  slack,
+  splunk,
+  testSelection,
+} from "./schemaFields";
+
+export const getFormSchema = (): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as const,
+    properties: {
+      jira: {
+        type: "object" as const,
+        title: "JIRA",
+        properties: {
+          ...jira.schema,
+        },
```

Comment:
No spread here and below
```suggestion
        properties: jira.schema,
```